pubtools/_quay/signature_handler.py

```diff
@@ -503,14 +503,16 @@ def sign_container_images(self, push_items):
 class OperatorSignatureHandler(SignatureHandler):
     """Class for handling the signing of index images."""
 
-    def construct_index_image_claim_messages(self, index_image, tag, signing_keys):
+    def construct_index_image_claim_messages(self, index_image, tag, stamp_tag, signing_keys):
```

Comment:
I think it would be better to allow list to be passed to "tags" parameter, instead of having two distinct parameters for tags. That way, when reading the method, the user doesn't have to wonder what the heck is a stamp_tag.